charts/incubator/shlink/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "2.10.1"
+appVersion: "v2.10.1"
```

Comment:
the questions.yaml error has nothing to do with chart.yaml.

Reply:
Actually I stand corrected.

Reply:
App version should NEVER have a `v`

Reply:
It's important to note that the output in release-tests are done AFTER overriding this variable.